lib/vrl/stdlib/src/chunks.rs

```diff
@@ -0,0 +1,134 @@
+use ::value::Value;
+use vrl::prelude::*;
+
+fn chunks(value: Value, chunk_size: Value) -> Resolved {
+    let bytes = value.try_bytes()?;
+    let chunk_size = chunk_size.try_integer()?;
+
+    if chunk_size < 1 {
+        return Err(r#""chunk_size" must be at least 1 byte"#.into());
+    }
+
+    if let Ok(chunk_size) = usize::try_from(chunk_size) {
+        Ok(bytes.chunks(chunk_size).collect::<Vec<_>>().into())
+    } else {
+        Err(r#""chunk_size" must be a valid usize for this target architecture"#.into())
```

Comment:
The term `usize` may be confusing to VRL users who aren't familiar with Rust. Maybe re-word this to just say the value is out of the valid range (and maybe calculate / display the actual valid range)